language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/common/constants/CommandConstants.java

```diff
@@ -129,4 +129,6 @@
     public static final String OPTIMIZE_IMPORTS_TITLE = "Optimize all imports";
     
     public static final String REPORT_USAGE_STATISTICS_COMMAND_TITLE = "Report usage statistics";
+    
+    public static final String MAKE_MAIN_FUNCTION_PUBLIC = "Make 'main' function public";
```

Comment:
What if we use `Convert to public function`? @IMS94 @gayalkuruppu @mohanvive  WDYT?

Reply:
+1

Reply:
> What if we use `Convert to public function`? @IMS94 @gayalkuruppu @mohanvive WDYT?

```suggestion
    public static final String CONVERT_FUNC_PUBLIC = "Convert to public function";
```
or

```suggestion
    public static final String MAKE_MAIN_FUNCTION_PUBLIC = "Convert to public function";
```
Should I change the Variable name as well? BTW we only make the **main function public**

Reply:
+1 to change the variable name as well